recipes/cccl/all/conanfile.py

```diff
@@ -18,50 +22,55 @@ class CcclConan(ConanFile):
         "muffle": True,
         "verbose": False,
     }
-    settings = "compiler",
 
-    _source_subfolder = "source_subfolder"
+    @property
+    def _cccl_dir(self):
+        return os.path.join(self.package_folder, "bin")
 
     def configure(self):
-        if self.settings.compiler != "Visual Studio":
-            raise ConanInvalidConfiguration("This recipe support only Visual Studio")
+        if not is_msvc(self):
+            raise ConanInvalidConfiguration("This recipe only supports msvc/Visual Studio.")
         del self.settings.compiler
 
+    def layout(self):
+        basic_layout(self, src_folder="source_subfolder")
```

Comment:
```suggestion
        basic_layout(self, src_folder="src")
```
First because is a pattern followed on CCI, second is to avoid max path length in Windows when building the package.